composer.json

```diff
@@ -30,78 +30,78 @@
         "laminas-skeleton-installer": [
             {
                 "name": "laminas/laminas-developer-tools",
-                "constraint": "^2.0.1",
+                "constraint": "^2.1.1",
                 "prompt": "Would you like to install the developer toolbar?",
                 "module": true,
                 "dev": true
             },
             {
                 "name": "laminas/laminas-cache",
-                "constraint": "^2.9.0",
+                "constraint": "^2.11.1",
                 "prompt": "Would you like to install caching support?",
                 "module": true
             },
             {
                 "name": "laminas/laminas-db",
-                "constraint": "^2.10.0",
+                "constraint": "^2.12.0",
                 "prompt": "Would you like to install database support (installs laminas-db)?",
                 "module": true
             },
             {
                 "name": "laminas/laminas-mvc-form",
-                "constraint": "^1.0",
+                "constraint": "^1.1.0",
                 "prompt": "Would you like to install forms support?",
                 "module": true
             },
             {
                 "name": "laminas/laminas-json",
-                "constraint": "^3.1.2",
+                "constraint": "^3.2",
                 "prompt": "Would you like to install JSON de/serialization support?"
             },
             {
                 "name": "laminas/laminas-log",
-                "constraint": "^2.11",
+                "constraint": "^2.13.1",
                 "prompt": "Would you like to install logging support?",
                 "module": true
             },
             {
-                "name": "laminas/laminas-mvc-console",
-                "constraint": "^1.2.0",
-                "prompt": "Would you like to install MVC-based console support? (We recommend migrating to zf-console, symfony/console, or Aura.CLI)",
+                "name": "laminas/laminas-cli",
+                "constraint": "^1.1.1",
+                "prompt": "Would you like to install command-line interface support?",
                 "module": true
```

Comment:
Checking `laminas-cli`. It does not provide or need module
```suggestion
```

Reply:
removed https://github.com/laminas/laminas-mvc-skeleton/pull/39/commits/1dca5cf6ed067f0758c2c4455f51c8c5c8794eac